features/support/world.js

```diff
@@ -14,11 +14,21 @@ class CustomWorld {
     this.exceptionScenario = false;
   }
 
+  createWindow(url, outputConsole) {
+    const virtualConsole = new VirtualConsole();

```

Comment:
I made this change with the virtualConsole to help clean up the test logs a bit.  Basically when we use this function here: https://github.com/cerner/cucumber-forge-report-generator/pull/64/files#diff-6605869b51324cd7403d348bda4f58fcef198b8edcffd1d7f3a9040c65bc51d5R69 to create a new window for a report with a particular hash location the report will automatically try to `scrollIntoView` the linked scenario/feature.  However, the headless Jsdom testing framework we are using does not natively support `scrollIntoView`.  In other places we are able to mock the function before it is called so that things are properly handled, but in this case I cannot access dom elements without the window and I cannot create the window without it running the `init` script which is what triggers the `scrollIntoView` call.  

When a scenario calls `createWindow` without providing an outputConsole param, then all the output from jsdom will get sent to this virtualConsole and will not be printed to the node console.  This keeps the errors around the undefined `scrollIntoView`  function from cluttering up the logs.  (Also, to be clear, the scenarios that call `createWindow` without providing an outputConsole param do not really need to test the scrollIntoView function so I think this is an acceptable tradeoff.)